index.html

```diff
@@ -1275,13 +1317,79 @@ <h2>DID Subject</h2>
 }
       </pre>
 
-      <p class="note">
+      <p class="note" title="Intermediate representations">
 <a>DID method</a> specifications can create intermediate representations of a
-<a>DID document</a> that do not contain the <code>id</code> property, such as
-when a <a>DID resolver</a> is performing <a>DID resolution</a>. However, the
-fully resolved <a>DID document</a> always contains a valid <code>id</code>
-property. The value of <code>id</code> in the resolved <a>DID document</a> is
-expected to match the <a>DID</a> that was resolved.
+<a>DID document</a> that do not contain the <code><a>id</a></code> property,
+such as when a <a>DID resolver</a> is performing <a>DID resolution</a>.
+However, the fully resolved <a>DID document</a> always contains a valid
+<code><a>id</a></code> property. The value of <code><a>id</a></code> in the
+resolved <a>DID document</a> is expected to match the <a>DID</a> that was
+resolved.
+      </p>
+    </section>
+
+    <section>
+      <h2>
+Control
+      </h2>
+
+      <p>
+Authorization is the mechanism used to state how operations are performed
+on behalf of the <a>DID subject</a>. A <a>DID controller</a> is authorized to
+make changes to the respective <a>DID document</a>.
+      </p>
+
+      <p>
+A DID document MAY include a <code><a>controller</a></code> property to
+indicate the <a>DID controller(s)</a>. If so:
+      </p>
+      <dl>
+          <dt><dfn>controller</dfn></dt>
+          <dd>
+The value of the <code>controller</code> property MUST be an
+<a data-cite="INFRA#ordered-set">ordered set</a> of <a
```

Comment:
The examples permit "controller" to be a string as well -- and many implementations do this; I think we should keep it.

Reply:
Updated with that suggestion